content/items/pyVHDLModel.md

```diff
@@ -11,11 +11,20 @@ tags: [
   "dom",
   "language-model",
   "python",
-  "VHDL",
+  "vhdl",
```

Comment:
Tags are not case sensitive. No neeed to fix it, but it's ok.

Reply:
ok